vtex/address-form
# Fix Guatemala address rules

#### What is the purpose of this pull request?

Fix typos in some states/cities from Guatemala, as described in the ticket [#238797](https://vtexhelp.zendesk.com/agent/tickets/238797) (Zendesk).

#### How should this be manually tested?

You can use the cart below to check the address-form both via Shipping Preview or Omnishipping for the updated regions.

- https://gtm--vtexgame1.myvtex.com/checkout/cart/add?sku=312&qty=1&seller=1&redirect=true&sc=2

#### Types of changes
